src/language-js/printer-estree.js

```diff
@@ -3979,7 +3972,7 @@ function printArgumentsList(path, options, print) {
   //    )
   // here, but not
   //    process.stdout.pipe(socket)
```

Comment:
We might want to change this comment now

Reply:
Moved and updated comment (let me know if you think no comment is necessary)